src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,89 @@
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+          </th>
+          <th>Title</th>
+          <th> </th>
+        </tr>
+      </thead>
+      <tbody>
+        {todos &&
+          todos.map(({ id, title, completed, userId }) => {
+            return (
+              <tr
+                data-cy="todo"
+                className={classNames({
+                  'has-background-info-light':
+                    choseTodo?.highlightedTodo && choseTodo.id === id,
+                })}
+                key={id}
+              >
+                <td className="is-vcentered">{id}</td>
+                <td className="is-vcentered">
+                  {completed && (
+                    <span className="icon" data-cy="iconCompleted">
+                      <i className="fas fa-check" />
+                    </span>
+                  )}
+                </td>
+                <td className="is-vcentered is-expanded">
+                  <p
+                    className={
+                      completed ? 'has-text-success' : 'has-text-danger'
+                    }
+                  >
+                    {title}
+                  </p>
+                </td>
+                <td className="has-text-right is-vcentered">
+                  <button
+                    data-cy="selectButton"
+                    className="button"
+                    type="button"
+                    onClick={() =>
+                      handleShowModal(userId, {
+                        title,
+                        id,
+                        completed,
+                        highlightedTodo: true,
+                      })
```

Comment:
```suggestion
                      handleShowModal(userId, {
                        title,
                        id,
                        completed,
                        highlightedTodo: true,
                      })
```
```suggestion
                      handleShowModal(todo)
```
just save the todo here